tools/Google.Cloud.Tools.SbomGenerator/Program.cs

```diff
@@ -37,25 +37,23 @@ public static void Main(string[] args)
             return;
         }
         string nupkgPath = args[0];
-        WriteSpdx(nupkgPath);
+        await WriteSpdxAsync(nupkgPath);
     }
 
     /// <summary>
     /// Examines a .nupkg file and writes an SPDX SBOM json file next to it.
     /// </summary>
     /// <param name="nupkgPath">Path to .nupkg file.</param>
-    /// <returns>Path to generated SPDX SBOM json file.</returns>
-    private static void WriteSpdx(string nupkgPath)
+    private static async Task WriteSpdxAsync(string nupkgPath)
```

Comment:
Any particular reason for going to an async model? It feels like it's making things a little more complex without a benefit - but I'm probably missing something.